### DISCUSSION TITLE
Header1 as a title, what if it is missing?

### DISCUSSION THREAD

**Discussion by webwesen** [zk-org/zk-nvim#24] [Help] [answered]
1st of all - a great piece of software, really impressive.

my use case - I have imported (basically copied into a zk vault) a lot of my MD notes, and indexed them. 
the plugin expects notes to have #-level heading to display it in Telescope lists, for example.
I mostly have just indented bullet lists...

is there some setting in config that would look at a file name instead?
Otherwise, I will have to add a header to all my files...

thanks!

**Comment by mickael-menu** [comment]
`zk` supports [custom line formats](https://github.com/mickael-menu/zk/blob/main/docs/tool-fzf.md#line-format) for the `fzf` mode, something similar could be added to `zk-nvim` eventually.

But as a quick workaround, I think we should fall back on the `path` if the `title` is empty. This is the default behavior of `zk list --interactive` as well.

**Reply by karimknaebel** [reply]
[fallback to `path` ](https://github.com/mickael-menu/zk-nvim/pull/25)

**Reply by mickael-menu** [reply]
Thanks! It's merged in `main`.

**Comment by karimknaebel** [comment, marked as answer]
@webwesen should be fixed now, can you confirm?

**Reply by webwesen** [reply]
confirming - works!
many thanks
